source/retryable-writes/tests/README.rst

```diff
@@ -327,10 +327,45 @@ and sharded clusters.
    in use MAY skip this test for sharded clusters, since ``mongos`` does not report
    this information in its ``serverStatus`` response.
 
+#. Test that drivers properly retry after encountering PoolClearedErrors. This
+   test MUST be implemented by any driver that implements the CMAP
+   specification.
+
+   1. Create a client with maxPoolSize=1 and retryWrites=true. If testing
+      against a sharded deployment, be sure to connect to only a single mongos.
+
+   2. Enable the following failpoint::
+
+       {
+           configureFailPoint: "failCommand",
+           mode: { times: 1 },
+           data: {
+               failCommands: ["insert"],
+               errorCode: 91,
+               blockConnection: true,
+               blockTimeMS: 1000
+           }
+       }
+
+   3. Start two threads and attempt to perform an ``insertOne`` simultaneously on both.
+
+   4. Verify that both ``insertOne`` attempts succeed.
+
+   5. Via CMAP monitoring, assert that the first check out succeeds.
+
+   6. Via CMAP monitoring, assert that the second check out fails due to a
```

Comment:
Same comments as earlier on the retryable reads prose test:

 * Should we expect a certain number of CommandStartedEvents?
 * Should we expect a PoolClearedEvent?